bigquery/snippets/snippet_test.go

```diff
@@ -72,36 +72,70 @@ func TestAll(t *testing.T) {
 		t.Errorf("listDatasets: %v", err)
 	}
 
-	tableID := fmt.Sprintf("golang_example_table_%d", time.Now().Unix())
-	if err := createTable(client, datasetID, tableID); err != nil {
-		t.Errorf("createTable(dataset:%q  table:%q): %v", datasetID, tableID, err)
+	tblInferredSchema := fmt.Sprintf("golang_example_table_inferred_%d", time.Now().Unix())
```

Comment:
Drop the `tbl` prefix? You might even be able to shorten them to `inferred`, `explicit`, and `empty`.

Reply:
removed the prefix, and simplified variables down to the suggested naming.